net-misc/krb5/krb5-1.20.recipe

```diff
@@ -0,0 +1,138 @@
+	"
+
+PROVIDES="
+	krb5$secondaryArchSuffix = $portVersion
+
+	cmd:compile_et$secondaryArchSuffix
+	cmd:gss_client$secondaryArchSuffix
+	cmd:gss_server$secondaryArchSuffix
+	cmd:k5srvutil$secondaryArchSuffix
+	cmd:kadmin$secondaryArchSuffix
+	cmd:kadmin.local$secondaryArchSuffix
+	cmd:kadmind$secondaryArchSuffix
+	cmd:kdb5_util$secondaryArchSuffix
+	cmd:kdestroy$secondaryArchSuffix
+	cmd:kinit$secondaryArchSuffix
+	cmd:klist$secondaryArchSuffix
+	cmd:kpasswd$secondaryArchSuffix
+	cmd:kprop$secondaryArchSuffix
+	cmd:kpropd$secondaryArchSuffix
+	cmd:kproplog$secondaryArchSuffix
+	cmd:krb5_config$secondaryArchSuffix
+	cmd:krb5_send_pr$secondaryArchSuffix
+	cmd:krb5kdc$secondaryArchSuffix
+	cmd:kswitch$secondaryArchSuffix
+	cmd:ktutil$secondaryArchSuffix
+	cmd:kvno$secondaryArchSuffix
+	cmd:sclient$secondaryArchSuffix
+	cmd:sim_client$secondaryArchSuffix
+	cmd:sim_server$secondaryArchSuffix
+	cmd:sserver$secondaryArchSuffix
+	cmd:uuclient$secondaryArchSuffix
+	cmd:uuserver$secondaryArchSuffix
+
+	lib:libcom_err$secondaryArchSuffix = 3.0
+	lib:libgssapi_krb5$secondaryArchSuffix = 2.2
+	lib:libgssrpc$secondaryArchSuffix = 4.2
+	lib:libk5crypto$secondaryArchSuffix = 3.1
+	lib:libkadm5clnt$secondaryArchSuffix = 12.0
+	lib:libkadm5clnt_mit$secondaryArchSuffix = 12.0
+	lib:libkadm5srv$secondaryArchSuffix = 12.0
+	lib:libkadm5srv_mit$secondaryArchSuffix = 12.0
+	lib:libkdb5$secondaryArchSuffix = 10.0
+	lib:libkrad$secondaryArchSuffix = 0.0
+	lib:libkrb5$secondaryArchSuffix = 3.3
+	lib:libkrb5support$secondaryArchSuffix = 0.1
+	lib:libverto$secondaryArchSuffix = 0.0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	krb5${secondaryArchSuffix}_devel = $portVersion
+	devel:libcom_err$secondaryArchSuffix = 3.0
+	devel:libgssapi_krb5$secondaryArchSuffix = 2.2
+	devel:libgssrpc$secondaryArchSuffix = 4.2
+	devel:libk5crypto$secondaryArchSuffix = 3.1
+	devel:libkadm5clnt$secondaryArchSuffix = 12.0
+	devel:libkadm5clnt_mit$secondaryArchSuffix = 12.0
+	devel:libkadm5srv$secondaryArchSuffix = 12.0
+	devel:libkadm5srv_mit$secondaryArchSuffix = 12.0
+	devel:libkdb5$secondaryArchSuffix = 10.0
+	devel:libkrad$secondaryArchSuffix = 0.0
+	devel:libkrb5$secondaryArchSuffix = 3.3
+	devel:libkrb5support$secondaryArchSuffix = 0.1
+	devel:libverto$secondaryArchSuffix = 0.0
+	"
+REQUIRES_devel="
+	krb5$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoreconf
+	cmd:awk
+	cmd:find
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	cmd:yacc
+	"
+
+TEST_REQUIRES="
+	cmd:python3
+	"
+
+BUILD()
+{
+	cd src
+	autoreconf
+	runConfigure ./configure
+	make -j$jobs
+}
+
+INSTALL()
+{
+	cd src
+
+	make install
+
+	mkdir -p $settingsDir/gss
+	cp $dataDir/examples/krb5/krb5.conf $settingsDir/krb5.conf
+	awk '{sub("/usr/local/var", "/var")}1' $dataDir/examples/krb5/kdc.conf > $localStateDir/krb5kdc/kdc.conf
+
+	fixPkgconfig
```

Comment:
I'm pretty sure this will error out when running haikuporter in strict mode, think you need to add `prepareInstalledDevelLibs` (and the names of the libraries) `before fixPkgconfig.`

Reply:
`prepareInstalledDevelLibs` breaks the build as the binaries included depend on the unversioned symlink of the libraries.

I've checked the recipe in strict mode (`haikuporter -S`) and it gave no errors.

Reply:
This probably means the libraries don't have a soname set or it is not set correctly. This can result in problems if the library ABI changes, it will not be possible to install multiple versions of it side-by-side.

If the soname is correctly set to the versionned name of the library, the binaries will not need the unversionned files anymore.

Reply:
Yeah, I can see the soname problem now.
Fixed it, will test a recipe with the new patchset later.

Reply:
Checked, everything is working.

I initially thought the linked name was the behavior of the project itself, but it was actually caused by a faulty patch...